### ISSUE TITLE
Create ColorManager

### ISSUE THREAD

**Issue by anirudh-eka** [GordeyChernyy/DrawingTool#3]
`DreamBrush` has data that `TriangleBrush` depends on. They should share a common service/resource that manages it; maybe a `ColorManager`...

**Comment by GordeyChernyy** [comment]
yeah, `DreamBrush` has swatches. `ColorManager` is a great idea. I'm going to create something like a color tool for creating swatches from images.
